src/css/wizard-navigation-bar.scss

```diff
@@ -133,127 +117,182 @@ $wz-param-indicator-color: null !global;
-      @include wz-state-circle($width, $height, $border-width);
+      $aw-param-indicator-state: 'hover' !global;
+      $aw-param-step-state: 'default' !global;
+      @include aw-circle($width, $height, $border-width);
       @content;
     }
     &.navigable.optional a:hover .step-indicator {
-      $wz-param-indicator-state: 'hover' !global;
-      $wz-param-indicator-color: $wz-color-optional !global;
+      $aw-param-indicator-state: 'hover' !global;
+      $aw-param-step-state: 'optional' !global;
       @content;
     }
     &.navigable.done a:hover .step-indicator {
-      $wz-param-indicator-state: 'hover' !global;
-      $wz-param-indicator-color: $wz-color-done !global;
+      $aw-param-indicator-state: 'hover' !global;
+      $aw-param-step-state: 'done' !global;
       @content;
     }
     &.navigable.current a:hover .step-indicator {
-      $wz-param-indicator-state: 'hover' !global;
-      $wz-param-indicator-color: $wz-color-current !global;
+      $aw-param-indicator-state: 'hover' !global;
+      $aw-param-step-state: 'current' !global;
       @content;
     }
     &.navigable.editing a:hover .step-indicator {
-      $wz-param-indicator-state: 'hover' !global;
-      $wz-param-indicator-color: $wz-color-editing !global;
+      $aw-param-indicator-state: 'hover' !global;
+      $aw-param-step-state: 'editing' !global;
       @content;
     }
     // 'completed' class takes priority
     // https://github.com/madoar/angular-archwizard/pull/221
     &.navigable.completed a:hover .step-indicator {
-      $wz-param-indicator-state: 'hover' !global;
-      $wz-param-indicator-color: $wz-color-done !global;
+      $aw-param-indicator-state: 'hover' !global;
+      $aw-param-step-state: 'done' !global;
       @content;
     }
   }
 }
 
 
-// Helper mixin to define indicator styles like 'small', 'large-filled-symbols' etc. for the given layout
-//
-// Arguments:
-//   $layout - 'horizontal' | 'vertical'
-//
-@mixin wz-define-styles($layout) {
-  &.small ul.steps-indicator {
-    @include wz-define-style($layout, $width: $wz-small-dot-width, $height: $wz-small-dot-height, $border-width: 0) {
-      @include wz-state-circle-with-background($wz-param-indicator-color);
-    }
-  }
-
-  &.large-filled ul.steps-indicator {
-    @include wz-define-style($layout, $border-width: 0) {
-      @include wz-state-circle-with-background($wz-param-indicator-color);
-    }
-  }
-
-  &.large-empty ul.steps-indicator {
-    @include wz-define-style($layout) {
-      @include wz-state-circle-with-border($wz-param-indicator-color);
-    }
-  }
-
-  &.large-filled-symbols ul.steps-indicator {
-    @include wz-define-style($layout, $border-width: 0) {
-      @include wz-state-circle-with-background-and-content($wz-param-indicator-color);
-    }
-  }
+// Define wizard styles specific to the configurated layouts (`$aw-layouts`) and indicator styles (`$aw-indicator-styles`)
+@mixin aw-define-styles() {
```

Comment:
Maybe add an empty line above the mixin comment to separate it better visually.

Reply:
But there are already two blank lines there. As far as I can see, three blank lines are never used anywhere in archwizard
s SCSS files. In my opinion, using three blank lines here would be inconsistent.